backend/app/adapter/routing/handle/search.go

```diff
@@ -34,9 +37,18 @@ type Filter struct {
 	Orders     []order.By
 }
 
+// SearchResponse represents the response to the Search API request.
+type SearchResponse search.Result
```

Comment:
I wonder whether the structure of search response can be directly specified here so that developers know what fields are expected. We may need a private constructor to convert `result` to `search response`.

Reply:
I will modify it and we can compare